.github/workflows/verify.yml

```diff
@@ -0,0 +1,34 @@
+name: Verify every day
+
+on:
+  schedule:
+    # every day at 9am
+    - cron: '0 9 * * *'
+
+env:
+  go-cache-name: go
+
+jobs:
+  verify:
+    name: Verify docs and spelling
+    runs-on: [ubuntu-latest]
+    steps:
+    - name: Set up Go 1.17
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.17
+    - name: Check-out code
+      uses: actions/checkout@v3
+    - name: Run verify scripts
+      run: make verify
+    - name: Checking for broken Markdown links for main branch
+      if: ${{ github.base_ref == 'main' }}
```

Comment:
I think this condition should be removed as well.

Reply:
removed, thanks.